stream/tag_mapper.js

```diff
@@ -71,21 +77,22 @@ module.exports = function(){
       // other names which we could use as the default.
       if( !doc.getName('default') ){
 
-        var defaultName =
-          _.get(tags, 'official_name') ||
-          _.get(tags, 'int_name') ||
-          _.get(tags, 'nat_name') ||
-          _.get(tags, 'reg_name') ||
-          doc.getName('en');
+        const defaultName = [
+          ...parseSemicolonDelimitedValues(_.get(tags, 'official_name')),
+          ...parseSemicolonDelimitedValues(_.get(tags, 'int_name')),
+          ...parseSemicolonDelimitedValues(_.get(tags, 'nat_name')),
+          ...parseSemicolonDelimitedValues(_.get(tags, 'reg_name')),
+          ...parseSemicolonDelimitedValues(doc.getName('en'))
+        ].filter(Boolean);
```

Comment:
More fanciness :P

So just to confirm, this is because you can't use `||` when assigning to `const`, yeah?

Might actually be a use case for `let` in that case, as it's much more clear what the intention is, plus each of these function calls has to always be evaluated, whereas chained `||` will have an early return as soon as there's a truthy value, right?